scripts/api-compatibility-check/breaking-api-check.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+print_usage() {
+        cat <<HELP_USAGE
+Usage:
+        $0 [-p MapboxMaps.zip]
+    -p  Path to pre-packaged MapboxMaps
+HELP_USAGE
+}
+
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+PACKAGER_DIR="$SCRIPT_DIR/../release/packager"
+
+API_DIGESTER_PATH="$(xcode-select -p)/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-api-digester"
+
+ARTIFACTS_ZIP_PATH=""
+
+main() {
+    local products_dir="$SCRIPT_DIR"
+    local product_artifacts_dir="$products_dir/artifacts"
+
+    rm -rf "$SCRIPT_DIR/API"
+    rm -rf "$product_artifacts_dir"
+
+    if [[ ! -f "$ARTIFACTS_ZIP_PATH" ]]; then
+        cd "$PACKAGER_DIR" || exit 1
+        ./package-mapbox-maps.sh
+
+        ARTIFACTS_ZIP_PATH="$PACKAGER_DIR/MapboxMaps.zip"
+    fi
+
+    unzip "$ARTIFACTS_ZIP_PATH" -d "$products_dir" >/dev/null
+
+    iOS_FRAMEWORK_PATHS=$(find "$product_artifacts_dir" -path '*.framework' ! -path "*simulator*" ! -path "*maccatalyst*")
```

Comment:
This feels a bit fragile. I wish we had a more robust way of locating the framework we want.

Reply:
Not sure why you feel it fragile. We are running `find` against MapboxMaps archive with the archive. We also have to modify each .framework to make swift-api-digester load modules successfully.
Can you share your view on how to make this part of the script less fragile?

Reply:
Maybe it'd be better to search *for* a specific framework instead of searching for the ones that don't match. I don't think it's worth changing for now.